### PR TITLE
ADD final NGSIv2 .apib

### PR DESCRIPTION
**Pull request by fgalan**
Github shows this PR as a file renaming. Actually, it is the addition of a new file and deletion of an old one (fiware-reference.apib, not needed anymore as the specification work has ended and stable and latest doc coverge).

### CODE REVIEW

File: doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib [hunk 1]
```diff
@@ -59,6 +113,11 @@ NGSI version 2 uses camel case (camelCase) syntax for naming properties and rela
 by the API. When  referring to URIs, as part of HATEOAS patterns, and to mark them appropriately,
 the suffix `_url` is added.
 
+## Reference Implementations
+
+* NGISv2 servers
```
Comment (jmcanterafonseca) [inline doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib]: NGSIv2 Context Brokers

File: doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib [hunk 2]
```diff
@@ -35,10 +35,64 @@ Sergio García Gómez (Telefónica I+D), Martin Bauer (NEC).
...
-https://lists.fiware.org/private/fiware-ngsi/ (list subscription required).
+This specification is the final and stable version of NGSIv2 API specification (v2.0).
+
+# Changelog
+
+Changes since RC-2018.07:
+
+* None
+
+Changes since RC-2008.04:
```
Comment (kzangeli) [inline doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib]: 2008? :)

Comment (fgalan) [reply doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib]: :O

File: doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib [hunk 3]
```diff
@@ -35,10 +35,64 @@ Sergio García Gómez (Telefónica I+D), Martin Bauer (NEC).
...
+
+Changes since RC-2017.11:
+
+* `orderBy` may include also `id` and `type` as ordering fields
+* Included `registrations_url` in `GET /v2` response
+* Added registration management operations
+   * `GET /v2/registrations`
+   * `POST /v2/registrations`
+   * `GET /v2/registrations/{id}`
+   * `PATCH /v2/registrations/{id}`
```
Comment (kzangeli) [inline doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib]: Did we implement PATCH?   Sorry, out-of-memory ...

File: doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib (hunk 1 above)
Comment (fgalan) [reply doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib]: Fixed in 0d64f7138b3462067571346ea6d71352131bfef0

File: doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib (hunk 2 above)
Comment (fgalan) [reply doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib]: Fixed in 45eb8eee53dd4193e688ec8d6e25b243611a7ba8

File: doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib [hunk 4]
```diff
@@ -35,10 +35,64 @@ Sergio García Gómez (Telefónica I+D), Martin Bauer (NEC).
...
+* None
+
+Changes since RC-2008.04:
+
+* Section "System/builtin Attributes" renamed to "Builtin Attributes"
+* Section "System/builtin Metadata" renamed to "Builtin Metadata"
+* Added new builtin attribute: `dateExpires`
+* Improved description in "Error Response" section
+* Added new option for `POST /v2/entities` operation: `upsert`
+* Clarifying about forwarding compliance in "Group Registrations"
```
Comment (kzangeli) [inline doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib]: Clarification about ...

File: doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib (hunk 3 above)
Comment (fgalan) [reply doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib]: No. It is part o the "Orion implementation gap", described at https://fiware-orion.readthedocs.io/en/master/user/ngsiv2_implementation_notes/index.html

File: doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib (hunk 4 above)
Comment (fgalan) [reply doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib]: Fixed in db5b589368f93c94cdc5d6b4cb24ee3a19398416

File: doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib (hunk 3 above)
Comment (kzangeli) [reply doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib]: Ah, ok.
What is "implemented" is the description in the doc :)

NTC

**Pull request merged**